doc/man1/flux-mini.rst

```diff
@@ -263,8 +263,10 @@ OTHER OPTIONS
    with priority in the range of 0 to 31 (default 16).
 
 **-v, --verbose**
-   *(run only)* Increase verbosity on stderr. For example, currently ``-v``
-   displays jobid, ``-vv`` displays job events, and ``-vvv`` displays exec events.
+   *(run,alloc only)* Increase verbosity on stderr. For example, currently
```

Comment:
should be `(run,alloc)`?  b/c there's two things instead of one?

Reply:
heh, good catch. I agree, thanks!